Convert non-string value to string (#7)
StudistCorporation/fluent-plugin-timestream: README.md

## Configuration
Please refer to the [sample config file](https://github.com/StudistCorporation/fluent-plugin-timestream/blob/main/fluent.conf.sample)

## Note

> If log file contains null value, plugin convert it to empty string.

Reviewer: ```suggestion
The plugin converts `nil` values in the log to empty string.
```
書いて思ったけど、実質nilになることってありえる？キーが存在しない時？

Author: ログファイルの中にjsonが入っていて、{ "key": null } となっている場合ですね。
Update ボタン押してしまったけど、json のことを話するなら nil じゃなくて null だった。修正します。。。

## Configuration
Please refer to the [sample config file](https://github.com/StudistCorporation/fluent-plugin-timestream/blob/main/fluent.conf.sample)

## Note
The plugin converts `null` values in the log to empty string.  

> e.g. `{key_name: null}` => `{key_name: ""}`

Author: 例も記載しておきました(意味あるかな。。。)
